developers/guides/managing-projects.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Managing Projects"
+description:
+  "Explore best practices for managing multiple environments in Livepeer Studio"
+---
+
+![Create a project](/images/project-creation.png)
+
+## Overview
+
+Projects in Livepeer Studio allow for the organization of streaming assets, API
```

Comment:
Terminology hygiene - "streaming assets" is slightly confusing, since we have "Streams" and "Assets". Maybe just "Streams, Assets"?